docs/dev/api/insights.md

```diff
@@ -1186,26 +1388,249 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+  ]
+}
+```
+
+</details>
+
+## Activity
+
+### Get Activity
+
+<details><summary><span className="api get">GET</span> <code>/rest/v1/users/&#123;user_id&#125;/activity</code></summary>
+<p/>
+
+Return daily statistics about test results and concurrency usage for the specified <code>user_id</code>.
+
+#### Parameters
+
+<table id="table-api">
+  <tbody>
+    <tr>
+     <td><code>user_id</code></td>
+       <td><p><small>| PATH | REQUIRED | STRING |</small></p><p>Return only jobs that belongs to the specified <code>user_id</code></p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>since</code></td>
+       <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>Return only jobs ran from the provided Unix timestamp on.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>until</code></td>
+       <td><p><small>| QUERY | OPTIONAL | STRING|</small></p><p>Return only jobs ran until the provided Unix timestamp.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>level</code></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>
+     Specifies the ownership level. Supported values are: <code>user</code>, <code>organization</code>
+     </p></td>
+    </tr>
+  </tbody>
+</table>
+
+<Tabs
+groupId="dc-url"
+defaultValue="us"
+values={[
+{label: 'United States', value: 'us'},
+{label: 'Europe', value: 'eu'},
+]}>
+
+<TabItem value="us">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request GET "https://api.us-west-1.saucelabs.com/rest/v1/users/<user_id>/activity" | json_pp
+```
+
+</TabItem>
+<TabItem value="eu">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request GET "https://api.eu-central-1.saucelabs.com/rest/v1/users/<user_id>/activity" | json_pp
+```
+
+</TabItem>
+</Tabs>
+
+#### Responses
+
+<table id="table-api">
+<tbody>
+  <tr>
+    <td><code>200</code></td>
+    <td colSpan='2'>Success.</td>
+  </tr>
+</tbody>
+<tbody>
+  <tr>
+    <td><code>422</code></td>
+    <td colSpan='2'>Validation Error.</td>
+  </tr>
+</tbody>
+</table>
+
+```jsx title="Sample Response"
+{
+  "ccy_exec_mean": [
+    0
+  ],
+  "ccy_exec_peak": [
+    0
+  ],
+  "datestamp": [
+    "2023-04-18"
+  ],
+  "jobs": [
+    0
+  ],
+  "minutes": [
+    0
+  ]
+}
+```
+
+</details>
+
+---
+
+### Get Activity for Users
+
+<details><summary><span className="api get">GET</span> <code>/rest/v1/users_activity</code></summary>
+<p/>
+
+Return daily statistics about test results and concurrency usage.
+
+#### Parameters
+
+<table id="table-api">
+  <tbody>
+    <tr>
+     <td><code>users</code></td>
+       <td><p><small>| QUERY | OPTIONAL | ARRAY of STRINGS |</small></p><p>Limit results to specific users.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>since</code></td>
+       <td><p><small>| QUERY | OPTIONAL | DATE |</small></p><p>Return only jobs ran from the provided Unix timestamp on.</p></td>
```

Comment:
```suggestion
       <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>Return only jobs ran after the provided Unix timestamp.</p></td>
```

Reply:
The descriptions for the `since` and `until` parameters in this section are categorized as a "DATE", whereas we describe the same parameters as a "STRING" in other places in this file.